updatemodules/partitions/modules/dualpartmodule/dualpartmodule_test.go

```diff
@@ -74,12 +74,11 @@ type testChecker struct {
  ******************************************************************************/
 
 var tmpDir string
-var testImage string
 
 var disk *testtools.TestDisk
 
 var stateController = testStateController{}
-var stateStorage = testStateStorage{state: []byte("")}
+var stateStorage = testStateStorage{state: []byte("{}")}
```

Comment:
What kind of warning is in this line?

Reply:
we have missed check error in getState. The state should be in json format. Empty string returns json parsing error.